### PR TITLE
added program to check orientation of 3 points

### PR DESCRIPTION
**Pull request by rutujaingole**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes!

### Description

This program is for checking the Orientation of three points in a 2-D plane in  C++

### Checklist

- [✓ ] I've read the contribution guidelines.
- [ ✓] I've checked the issue list before deciding what to submit.
- [ ✓] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#4070

### CODE REVIEW

File: C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+   Test Case 2:
+Sample Input :Input the desired values for point a, point b, point c to check their Orientation
+                  Input x and y values for point a
+                  0 0
+                  Input x and y values for point b
+                  4 4
+                  Input x and y values for point c
+                 -1 1
+ Sample Output : Orientation : Counter-Clockwise */
```
Comment (HarshCasper) [inline C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp]: Keep only one test case. You can remove it

File: C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    cout << "Input the desired values for point a, point b, point c to check their Orientation \n";  // To input the desired values
+    cout << "Input x and y values for point a" << endl;
+    cin >> x1 >> y1;
+    cout << "Input x and y values for point b" << endl;
+    cin >> x2 >> y2;
+    cout << "Input x and y values for point c." << endl;
+    cin >> x3 >> y3;
+
+
+    int result = CheckOrientation(x1, y1, x2, y2, x3, y3);                   //calling the function for checking the orientation
```
Comment (HarshCasper) [inline C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp]: Keep comments above the line of code. Not beside it

File: C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp [hunk 3]
```diff
@@ -0,0 +1,81 @@
+/* This Program is to check the Orientation of three given points on a 2-D plane
```
Comment (HarshCasper) [inline C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp:1]: Can you add a bit more description?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+     i.e, a) Counter-Clockwise
+          b) Clockwise
+          c) Collinear
+ Based on the slope, we can categorize the orientation of the three points.
+ If the slope of line segment (a,b)> slope of line segment (b,c), then the orientation is clockwise or right turn
+ and vice versa for counter - clockwise or left turn.
+ When the three points are collinear, the slope of both the lines are equal*/
+
+
+
```
Comment (HarshCasper) [inline C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp]: Remove extra lines of Code

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp (hunk 4 above)
Comment (rutujaingole) [reply C-Plus-Plus/computational_geometry/Check_Orientation_of_three_points.cpp]: @HarshCasper Removed the extra lines. Is the code fine now?

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**